src/Features/LanguageServer/Protocol/Features/UnifiedSuggestions/UnifiedSuggestedActionsSource.cs

```diff
@@ -519,6 +511,96 @@ private static UnifiedSuggestedActionSet OrganizeRefactorings(Workspace workspac
                 title: null,
                 priority: GetUnifiedSuggestedActionSetPriority(actions.Max(a => a.CodeActionPriority)),
                 applicableToSpan: refactoring.CodeActions.FirstOrDefault().applicableToSpan);
+
+            // Local functions
+            async Task<IUnifiedSuggestedAction> GetUnifiedSuggestedActionSetAsync(CodeAction codeAction, TextSpan? applicableToSpan, TextSpan selection, CancellationToken cancellationToken)
+            {
+                if (codeAction.NestedCodeActions.Length > 0)
+                {
+                    _ = ArrayBuilder<IUnifiedSuggestedAction>.GetInstance(codeAction.NestedCodeActions.Length, out var nestedActions);
```

Comment:
```suggestion
                    using var _ = ArrayBuilder<IUnifiedSuggestedAction>.GetInstance(codeAction.NestedCodeActions.Length, out var nestedActions);
```

Reply:
second time seeing this.  ArrayBuilders are pooled.  teh `using` ensures that we return the builder to the pool when done with it.